index.md

```diff
@@ -4,11 +4,12 @@ title: Orsa Scholis
 
 ## Orsa Scholis
 
-Än Website für eusi Schuelprojekt und anderi gemeinsami Bastlereie
+Än Website für eusi ([@lukasbischof](https://github.com/lukasbischof), [@TheFehr](https://github.com/TheFehr)) Schuelprojekt und anderi gemeinsami Bastlereie
```

Comment:
Please use "Ä Website", since "Website" is a feminine noun in german (https://www.duden.de/rechtschreibung/Website)